pbhmergers.py

```diff
@@ -221,7 +224,7 @@ def mergerhalflife(self,mass,threefac=True, bhmass=None):
             threefac = self.threebodyratio(mass)
             threefac = np.max([threefac, np.ones_like(threefac)],axis=0)
             rate *= threefac
-        return 0.5*(mass/bhmass)/rat
```

Comment:
I didn't take points off for this, but the better fix is to replace rat with rate